ui/public/config.json

```diff
@@ -18,6 +18,14 @@
     "404": "assets/404.png",
     "500": "assets/500.png"
   },
+  "externalLinksIcon": "",
+  "externalLinks": [
+    {
+      "title": "Example",
+      "link": "",
+      "icon": ""
+    }
+  ],
```

Comment:
@RodrigoDLopez is this needed now with your new changes?

Reply:
no, it's not. Thanks